core/appeal/service.go

```diff
@@ -78,6 +76,18 @@ type auditLogger interface {
 	Log(ctx context.Context, action string, data interface{}) error
 }
 
+type CreateAppealOption func(*createAppealOptions)
+
+type createAppealOptions struct {
+	IsAdditionalAppeal bool
+}
+
+func WithAdditionalAppealOption() CreateAppealOption {
```

Comment:
```suggestion
func CreateWithAdditionalAppeal() CreateAppealOption {
```
better to scoped it specific to `Create` action

Reply:
@mabdh can we just rely on the returned type `CreateAppealOption` instead of changing the func name?

Reply:
@mabdh makes sense. Made the change accordingly.